examples/ui.apps/src/main/content/jcr_root/apps/forms-components-examples/components/form/titlev2/.content.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You should not create separate proxy for v2, the existing proxy which exists should point to v2

Reply:
Implemented!